scripts/haas

```diff
@@ -14,5 +14,8 @@
 # express or implied.  See the License for the specific language
 # governing permissions and limitations under the License.
 
+import sys
+
 from haas import cli
-cli.main()
+
+sys.exit(cli.main())
```

Comment:
Good point.  Right now, ``cli.main`` calls ``sys.exit`` manually on errors, but not on normal results.  So we can just revert this part of the change.  (I copied it over from your PR without thinking about it very hard.)